enterprise/backend/test/metabase_enterprise/serialization/v2/extract_test.clj

```diff
@@ -1376,17 +1376,19 @@
                         (map serdes/path)
                         set))))
 
-          (testing "select a collection where a dashboard contains parameter's source is card from another collection"
-            (is (=? #{[{:model "Collection"    :id coll4-eid :label "forth_collection"}]
-                      [{:model "Dashboard"     :id dash4-eid :label "dashboard_4"}]
-                      [{:model "Card"          :id c4-eid  :label "question_4_1"}]
-                      ;; card that parameter on dashboard linked to
-                      [{:model "Card"          :id c1-1-eid  :label "question_1_1"}]
-                      ;; card that the card on dashboard linked to
-                      [{:model "Card"          :id c1-2-eid  :label "question_1_2"}]}
-                    (->> (extract/extract {:targets [["Collection" coll4-id]] :no-settings true :no-data-model true})
-                         (map serdes/path)
-                         set)))))))))
+          #_;; (sanya) NOTE: this looks like an irrelevant test? I mean if we do escape analysis and don't allow
+          ;; leaking cards from unrelated collections, why this test?
```

Comment:
I think it's badly phrased, but this test is still relevant.

Parameter on a dashboard gets its values from a card in another collection. That link should appear in `deps` and `descendants`.

Therefore:
- Selective export rejects it, since it's escaping.
- Global export is fine, and during import the `deps` work properly.

So I think this test is still important, though its expectation should be changed to an escape error. This is the gap called out in a TODO, which I highlighted with a comment.